treadmill/infra/setup/scripts/provision-zookeeper.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+
+#Install
+
+DOMAIN=$1
+
+if [ ! -e /etc/yum.repos.d/cloudera-cdh5.repo ]; then
+    curl -L https://archive.cloudera.com/cdh5/redhat/5/x86_64/cdh/cloudera-cdh5.repo?_ga=2.172934241.314812559.1496985621-1968320782.1496291714 -o /etc/yum.repos.d/cloudera-cdh5.repo
```

Comment:
probably ok, but preferably lets create a constant for clouder - ZK_REPO or something like that.